qhbmlib/infer/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 The QHBM Library Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Module for qhbmlib.infer.*"""
+
+from qhbmlib.infer.ebm import EnergyInferenceBase, EnergyInference, AnalyticEnergyInference, BernoulliEnergyInference
+from qhbmlib.infer.ebm_utils import probabilities
+from qhbmlib.infer.qhbm import QHBM
+from qhbmlib.infer.qhbm_utils import density_matrix, fidelity
```

Comment:
Wondering more about this, isn't use of `import *` frowned upon?  Are there use cases where wildcard imports are still the norm, which we would like to support?

Reply:
definitely frowned upon - the real reason TF does this is because it's a form of documentation in itself (tells the programmer which objects are a part of the public API) and useful for docs (can have them auto-generated to only describe objects marked public)

Reply:
Makes sense - defined `__all__` in the subpackage `__init__` files